modules/claims_api/config/schemas/526.json

```diff
@@ -294,7 +293,33 @@
               ],
               "properties": {
                 "homelessSituationType": {
-                  "type": "string"
+                  "type": "string",
+                  "default": "OTHER",
+                  "enum": ["FLEEING_CURRENT_RESIDENCE",
```

Comment:
Correct me if I am wrong here, but I thought the point of this was to use the translation keys here https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/disability_compensation_form/data_translation_all_claim.rb#L13 and not the values?

Reply:
Fixed